yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/core_abilities.dm

```diff
@@ -0,0 +1,364 @@
+	if(HAS_TRAIT(target, TRAIT_DARKSPAWN_DEVOURED))
+		self_text += span_warning("[target]'s mind is already damaged by previous devouring and has granted less willpower and no lucidity.")
+	else
+		self_text += span_velvet("This individual's lucidity brings you one step closer to the sacrament...")
+		self_text += span_warning("After meddling with [target]'s mind, they will grant less willpower and no lucidity any future times their will is devoured.")
+	self_text += span_warning("[target] is now severely weakened and will take some time to recover.")
+	caster.visible_message(span_warning("[caster] gently lowers [target] to the ground..."), self_text.Join("<br>"))
+
+	//pass out the willpower and lucidity to the darkspawns
+	var/datum/team/darkspawn/team = darkspawn.get_team()
+	if(team)
+		team.grant_willpower(willpower_amount)
+		team.grant_lucidity(lucidity_amount)
+
+	//apply the long-term debuffs to the victim
+	target.apply_status_effect(STATUS_EFFECT_BROKEN_WILL)
+	target.apply_status_effect(STATUS_EFFECT_DEVOURED_WILL)
+	ADD_TRAIT(target, TRAIT_DARKSPAWN_DEVOURED, type)
+	return TRUE
+
+//////////////////////////////////////////////////////////////////////////
+//--------------------------Glorified handcuffs-------------------------//
+//////////////////////////////////////////////////////////////////////////
+/datum/action/cooldown/spell/touch/restrain_body
+	name = "Restrain body"
+	desc = "Forms rudimentary restraints on a target's hands."
+	panel = "Darkspawn"
+	button_icon = 'yogstation/icons/mob/actions/actions_darkspawn.dmi'
+	sound = null
+	background_icon_state = "bg_alien"
+	overlay_icon_state = "bg_alien_border"
+	buttontooltipstyle = "alien"
+	button_icon_state = "restrain_body"
+	check_flags = AB_CHECK_HANDS_BLOCKED |  AB_CHECK_IMMOBILE | AB_CHECK_LYING | AB_CHECK_CONSCIOUS
+	spell_requirements = SPELL_REQUIRES_HUMAN
+	invocation_type = INVOCATION_NONE
+	hand_path = /obj/item/melee/touch_attack/darkspawn
+	var/tying = FALSE //If we're tying someone's hands
+
+/datum/action/cooldown/spell/touch/restrain_body/can_cast_spell(feedback)
+	if(tying)
+		return
+	return ..()
+
+/datum/action/cooldown/spell/touch/restrain_body/is_valid_target(atom/cast_on)
+	return iscarbon(cast_on)
+
+/datum/action/cooldown/spell/touch/restrain_body/cast_on_hand_hit(obj/item/melee/touch_attack/hand, mob/living/carbon/target, mob/living/carbon/caster)
+	var/datum/antagonist/darkspawn/darkspawn = isdarkspawn(caster)
+	if(!darkspawn || tying || target == caster) //no tying yourself
+		return
+	if(is_darkspawn_or_thrall(target))
+		to_chat(caster, span_warning("You cannot restrain allies."))
+		return
+	if(!istype(target))
+		to_chat(caster, span_warning("[target]'s mind is too pitiful to be of any use."))
+		return
+	if(target.handcuffed)
+		to_chat(caster, span_warning("[target] is already restrained."))
+		return
+
+	to_chat(caster, span_velvet("Koce ra..."))
+	to_chat(caster, span_velvet("You begin restraining [target]..."))
```

Comment:
Would these be better as balloon alert (except the "Koce ra" one)

Reply:
i'll change them all to balloon alerts